server/index.js

```diff
@@ -94,12 +155,83 @@ io.on('connection', socket => {
 });
 
 /* Routes */
+app.get('/logout', (req, res) => {
+  console.log('mysession', req.session);
+  if (req.session.userName){
+    delete req.session.userName;
+  }
+  req.logout();
+  console.log('mysession after logout', req.session);
+  res.send('N/A!');
```

Comment:
Please send back to the client, an appropriate status code. Something like `200` for this case. Here's a good resource for server-side status codes: [HTTP Status Codes](http://www.restapitutorial.com/httpstatuscodes.html)